lib/authorizer/oauth2.js

```diff
@@ -13,6 +13,8 @@ var _ = require('lodash'),
     ];
 
 /**
+ * OAuth-2 does not require to maintain it's state.
```

Comment:
Can you drop all the “does not require”?! And only add the one where it’s required and with a bit more explanation. :-p

Reply:
sure :)